package.json

```diff
@@ -35,6 +35,7 @@
         "ts-morph": "^22.0.0"
     },
     "devDependencies": {
+        "eslint-plugin-prettier": "^4.2.1",
```

Comment:
Unsure why this dependency is needed now, but Lint is complaining that it is missing this dependency:

<img width="2104" alt="Screenshot 2025-01-27 at 09 54 13" src="https://github.com/user-attachments/assets/d6b24d2d-2451-49b6-8162-d7c1dee969cd" />

Reply:
It isn't needed in main 🤔 Maybe revert your changes to package-lock.json to main and try it again?

Reply:
true, removed it.